config/ur10e_controllers.yaml

```diff
@@ -1,6 +1,6 @@
 # Settings for ros_control control loop
 hardware_control_loop:
-   loop_hz: 500
+   loop_hz: 100
```

Comment:
note: an e-Series controller runs internally at 500 Hz.

Lowering this value could have undesirable consequences.

Reply:
thank you for this insight - developers had been seeing the following error repeatedly, which suggests a communicational mismatch: 
"[ERROR] [1651696192.097982709]: Pipeline producer overflowed! <RTDE Data Pipeline>" 
This has been removed from the merge request edits

Reply:
IIRC, upstream has fixed this some time ago.

Running at a lower rate is only a work-around, not a solution.

I'd suggest to make sure you're using an up-to-date version of the `ur_robot_driver`.